ui/src/components/TrafficPage.tsx

```diff
@@ -327,10 +362,11 @@ export const TrafficPage: React.FC<TrafficPageProps> = ({onTLSDetected, setAnaly
                     </div>
                 </div>
                 <div className={classes.details}>
-                    {selectedEntryData && <EntryDetailed entryData={selectedEntryData} updateQuery={updateQuery}/>}
+                    {selectedEntryData && <EntryDetailed entryData={selectedEntryData} updateQuery={updateQuery} />}
                 </div>
             </div>}
-            {tappingStatus && <StatusBar tappingStatus={tappingStatus}/>}
+            {tappingStatus && <StatusBar tappingStatus={tappingStatus} />}
+            {serviceMapModalOpen && <ServiceMapModal isOpen={serviceMapModalOpen} onOpen={() => setServiceMapModalOpen(true)} onClose={() => setServiceMapModalOpen(false)} />}
```

Comment:
is the button shown in UI if the feature flag is disabled?

Reply:
no, it will be hidden based on the "serviceMapStatus" value
https://github.com/up9inc/mizu/pull/623/files#diff-7170980be6ad275bd973945378d09e7705f54e48edd42e2daac359d7d9793c15R312